css/register_account_details.css

```diff
@@ -0,0 +1,76 @@
+body,
+html {
+  height: 100%;
+  overflow: hidden;
+}
+.toasts {
+  position: fixed;
+  right: 0;
+  left: 0;
+  top: 1%;
+  margin-right: auto;
+  margin-left: auto;
+  min-height: 10em;
+  width: 100%;
+}
+.register_image {
+  background-image: url("../img/background.jpg");
+  filter: blur(8px);
+  -webkit-filter: blur(8px);
+  height: 100%;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.register_form {
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(152, 149, 158, 0.25);
+  color: white;
+  font-weight: bold;
+  border-radius: 5px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 2;
+  min-width: 450px;
+  padding: 15px;
+  margin: auto;
+  text-align: center;
+}
+
+.register_form .checkbox,
+label,
+a {
+  font-weight: 400;
+}
+
+.register_form .form-floating:focus-within {
+  z-index: 2;
+}
+
+.show-hide-pwd-eye {
+  margin: 15px 0 0 0;
+  padding: 5px 10px 5px 10px;
+  border-radius: 0px 5px 5px 0px;
+  height: 38px;
+  bottom: 53px;
+  background-color: rgba(211, 211, 211, 0.5);
+  float: right;
+  position: relative;
+  z-index: 5;
+
+  cursor: pointer;
+}
+
+.show-hide-pwd-eye:active {
+  padding: 8px 10px 5px 10px;
+}
+
+.align-center {
+  position: relative;
+  align-items: center;
+  left: 25px;
+  top: 15px;
+}
```

Comment:
There are a lot of duplications with my css code. Maybe we should discuss about creating a global css file for things everyone uses quite often?